index.html

```diff
@@ -14,8 +14,51 @@
 
 <div class="my-container">
   
-    <img src="logo.png" width= 400>
+  <img src="logo.png" width="400">
   <div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <div class="carousel-navbar">
```

Comment:
I moved this `<div>` up here and removed a few extra `<div>`'s that you didn't need.